crates/host/src/wasmbus/mod.rs

```diff
@@ -211,7 +240,13 @@ struct Handler {
     origin: WasmCloudEntity,
     // package -> target -> entity
     links: Arc<RwLock<HashMap<String, HashMap<String, WasmCloudEntity>>>>,
-    targets: Arc<RwLock<HashMap<TargetInterface, TargetEntity>>>,
+    /// The current link name to use for interface targets, overridable in actor code via set_target()
+    interface_link_name: Arc<RwLock<String>>,
+    // link name -> package -> interface -> target
+    #[allow(clippy::type_complexity)]
+    interface_links: Arc<RwLock<HashMap<String, HashMap<String, HashMap<String, String>>>>>,
```

Comment:
Nit: This clippy lint is actually right here, this is a pain to read so it might be good to alias

Reply:
For sure, I agree, I was just planning on aliasing / renaming once we have the type locked down since it might change if that makes sense.